### ISSUE TITLE
How to prepare the training dataset from custom dataset

### ISSUE THREAD

**Issue by rose-jinyang** [VITA-Group/FasterSeg#44]
Hello
How are you?
Thanks for contributing this project.
I am going to train a new model with my custom dataset (supervisely-person-dataset).
How should I prepare the training dataset?
Thanks

**Comment by chenwydj** [comment]
Hi @rose-jinyang! Thank you for your interest in our work!

To train on your own dataset:
1. Prepare your RGB images and ground truth label (for file format please refer to [Cityscapes](https://www.cityscapes-dataset.com/downloads/)).
2. Prepare file name lists as "dataset files" for data loading (see [examples here](https://github.com/VITA-Group/FasterSeg/tree/master/tools/datasets/cityscapes)).
3. Change `num_classes` to the semantic classes in your data (in [configuration file](https://github.com/TAMU-VITA/FasterSeg/blob/master/search/config_search.py#L40)).
